backend/src/packages/users/user.repository.ts

```diff
@@ -37,6 +37,17 @@ class UserRepository implements IRepository {
     return result[0] ? UserEntity.initialize(result[0]) : null;
   }
 
+  public async findByEmail(value: string): Promise<UserEntity | null> {
+    const result = await this.db
+      .driver()
+      .select()
+      .from(this.usersSchema)
+      .where(eq(this.usersSchema.email, value.toLowerCase()))
+      .execute();
+
+    return result[0] ? UserEntity.initialize(result[0]) : null;
```

Comment:
I know, we discussed another approach, but using magic numbers is still isn't good practice.
```
const [user = null] = await ...
return user ? 
```
This should fix error you mentioned in previous PR. And it's actually more accurate

Reply:
yes, it helped. Thank you!